plugins/hydra_nevergrad_sweeper/tests/test_nevergrad_sweeper_plugin.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+# noinspection PyUnresolvedReferences
+from typing import Any
+
+import nevergrad as ng  # type: ignore
+import pytest
+from omegaconf import DictConfig, OmegaConf
+
+from hydra.core.plugins import Plugins
+from hydra.plugins.sweeper import Sweeper
+from hydra.test_utils.test_utils import (  # noqa: F401
+    TSweepRunner,
+    chdir_plugin_root,
+    sweep_runner,
+)
+from hydra_plugins.hydra_nevergrad_sweeper import NevergradSweeper
+from hydra_plugins.hydra_nevergrad_sweeper.core import make_parameter
+
+chdir_plugin_root()
+
+
+def test_discovery() -> None:
+    # Tests that this plugin can be discovered via the plugins subsystem when looking at the Sweeper plugins
+    assert NevergradSweeper.__name__ in [x.__name__ for x in Plugins.discover(Sweeper)]
+
+
+@pytest.mark.parametrize(  # type: ignore
+    "string,param_cls,value_cls",
+    [
+        ("blu,blublu", ng.p.Choice, str),
+        ("0,1,2", ng.p.TransitionChoice, int),
+        ("0.0,12.0,2.0", ng.p.Choice, float),
+        ("1:12", ng.p.Scalar, int),
+        ("1.0:12", ng.p.Scalar, float),
+        ("blublu", str, str),
+        (
+            "Scalar(init=12.1).set_mutation(sigma=3).set_integer_casting()",
+            ng.p.Scalar,
+            int,
+        ),
+    ],
+)
+def test_make_parameter(string: str, param_cls: Any, value_cls: Any) -> None:
+    param = make_parameter(string)
+    assert isinstance(param, param_cls)
+    if param_cls is not str:
+        assert isinstance(param.value, value_cls)
+
+
+# pylint: disable=redefined-outer-name
```

Comment:
I am not using pylint in Hydra (at the moment) so I am not sure why you need this.

Reply:
For my editor...
You prefer I remove it?

Reply:
It's fine. just expect others not using the same editor to maybe introduce some new issues for you down the line.